R/expr.R

```diff
@@ -30,7 +30,7 @@
 #' @export
 #' @examples
 #' # expr_eval() works just like base::eval():
-#' env <- env_new(dict = list(foo = "bar"))
```

Comment:
we already have `as_env()`, which is indeed synonymous with `new_env()` in that case.